packages/rag/.env.example

```
@@ -0,0 +1,15 @@
+# Naver Cloud API
+CLOVASTUDIO_API_KEY=your_clova_api_key_here
+
+# Google Gemini API
+GEMINI_API_KEY=your_gemini_api_key_here
+
+# PostgreSQL Database
+DB_HOST=localhost
+DB_PORT=5432
+DB_NAME=your_database_name
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_NAME key should go before the DB_PORT key

(UnorderedKey)